web/styles/popovers.css

```diff
@@ -25,6 +25,20 @@
     }
 }
 
+.tippy-box[data-theme="popovers"] {
```

Comment:
This name seems a bit confusing -- we already have the `light-border` theme, above, which we use for our popovers; I think we probably want to call this something more like `dropdown-widget` or `typeahead-popovers` if that's a coherent name for this theme (will typeahead look kinda similar)?

Reply:
I think `dropdown-widget` sounds good. I am not sure about typeaheads.